File: stringMingling/stringMingling.py
def mingle(stringp,stringq):
    newString = ""
    for i in range(len(stringp)):
        newString += stringp[i] + stringq[i]
    return newString
    
stringp = "abcde"
stringq = "pqrst"
print(mingle(stringp,stringq))

File: stringMingling/stringMingling.java
package stringMingling;

public class stringMingling {
    
    public static String stringMing(String stringp, String stringq){
        String newString = "";
        for(int i=0; i<stringp.length();i++){
            newString += stringp.charAt(i);
            newString += stringq.charAt(i);
        }
        return newString;
    }

    public static void main(String[] args) {
        String stringp = "abcde";
        String stringq = "pqrst";
        System.out.println(stringMing(stringp, stringq));
    }
}
